app/compare/compare.css

```diff
@@ -171,19 +165,33 @@
   opacity: 1;
 }
 
-.compare-invocations .diff-line pre {
+.compare-invocations pre {
   padding: 0;
   margin: 0;
 }
 
-.compare-invocations .diff-line.removed {
-  color: #f44336;
+.compare-invocations del.removed {
+  background: #ffcdd2;
+  border-radius: 2px;
+  text-decoration: line-through;
```

Comment:
can we not strike through? adds a lot of visual noise - and i think the red probably says enough

Reply:
I did that for accessibility reasons - people with certain types of colorblindness (Deuteranopia or Achromatopsia) wouldn't be able to tell the difference between added and removed.

Reply:
FWIW this styling was basically copied from diff-match-patch's `diff_prettyHTML` function so I bet people at Google have thought through the practical concerns about the styling

Reply:
We could use `+` / `-` as visual indicators for accessibility instead, but this diff library that we're using now doesn't seem to have a way to diff on a line-by-line basis.

Reply:
The Github style + / - indicators which we used to have look a lot cleaner.

If you look at the examples in that repo - it was definitely created by engineers, not designers (and it hasn't been updated for 2-3 years).
https://neil.fraser.name/software/diff_match_patch/demos/diff.html

The biggest source of visual noise is the strikethrough whitespace.
<img width="120" alt="Screen Shot 2020-11-06 at 1 23 12 PM" src="https://user-images.githubusercontent.com/1704556/98416034-3a643a00-2033-11eb-80fb-52251ee850ca.png">

I could probably live with single line stike-throughs, but multi-line are pretty overboard.

Reply:
OK, I found a way to use the diff-match-patch library in line mode, and reverted back to the previous styling:

![image](https://user-images.githubusercontent.com/2414826/98426726-048d7880-2068-11eb-9628-681fe2301b76.png)